packages/lesswrong/lib/reviewUtils.tsx

```diff
@@ -9,16 +9,38 @@ const isLWForum = forumTypeSetting.get() === "LessWrong"
 
 export type ReviewYear = 2018 | 2019 | 2020 | 2021
 
+export function getReviewYearFromString(str) {
+  const year = parseInt(str)
```

Comment:
Catch error and explicitly return `undefined` if the user typed in something other than a year?